experiment_scheduler/submitter/log.py

```diff
@@ -0,0 +1,77 @@
+"""
+[TODO] exs status command Explanation
+"""
+import ast
+import grpc
+import argparse
+from os import path as osp
+from experiment_scheduler.common.settings import USER_CONFIG
+from experiment_scheduler.master.grpc_master import master_pb2
+from experiment_scheduler.master.grpc_master import master_pb2_grpc
+# from experiment_scheduler.common.logging import get_logger
+import logging
+
+
+_level_color_map = {
+    logging.INFO: "\u001b[32m [INFO] " + "%(message)s" + "\u001b[0m",
+    logging.WARNING: "\u001b[33m [WARN] " + "%(message)s" + "\u001b[0m",
+    logging.ERROR: "\u001b[31m [ERROR] " + "%(message)s" + "\u001b[0m",
+}
+
+def get_client_logger(name, task_id, *args, **kwargs):
+    logger = logging.getLogger(name=name, *args, **kwargs)
+    ch = logging.StreamHandler()
+    ch.setFormatter(ClientLoggingFormatter())
+    logger.addHandler(ch)
+    logger.setLevel(logging.INFO)
+    return logger
+
+
+class ClientLoggingFormatter(logging.Formatter):
+    """
+    custom logging
+    """
+
+    def format(self, record: logging.LogRecord) -> str:
+        log_fmt = _level_color_map.get(record.levelno)
+        formatter = logging.Formatter(log_fmt)
+        return formatter.format(record)
```

Comment:
We use logger instead of print.
Because we need new format, we made the new logger.